packages/cozy-stack-client/src/errors.js

```diff
@@ -8,6 +8,19 @@ export default {
   INVALID_TOKEN
 }
 
+const hasWwwAuthenticateExpiredTokenError = response => {
+  const invalidTokenRegex = /invalid_token/
```

Comment:
We should not mask this error and should not transform it in `expired token`. 

If you don't have the time to handle it correctly, I prefer that we do not transform it.

Reply:
I force-pushed a new version that returns `Expired token` only when the description contains the expired token description, and `Invalid token` when this description is not about expiration or if unset : https://github.com/cozy/cozy-client/compare/220a641ca974f0eef33a6f22826251071bdf9c63..3c7d3aea066bdfd2c93b1a731385f8fc52fed615